db/migrate/20231018141450_add_cpu_cores_columns_to_resource_pools.rb

```diff
@@ -0,0 +1,34 @@
+class AddCpuCoresColumnsToResourcePools < ActiveRecord::Migration[6.0]
+  class ResourcePools < ActiveRecord::Base
```

Comment:
The class should be `ResourcePool` (surprised this work without a `self.table_name`)
https://github.com/ManageIQ/manageiq/blob/master/app/models/resource_pool.rb#L1